.hooks/linters/mdstyle.rb

```diff
@@ -1,44 +0,0 @@
-################################################################################
-# Style file for markdownlint.
-#
```

Comment:
default editor auto-format settings at Meta produce markdown that is compatible with most rules - exceptions no longer required